docs/manage/ingestion-volume/ingest-budgets/minute-volume.md

```diff
@@ -0,0 +1,110 @@
+---
+id: minute-volume
+title: Minute Volume (Beta)
+description: Use Ingest Budgets Minute Volume to control data ingestion to minute granularity.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<!-- IMPORTANT: When this doc goes from Beta to GA, integrate this content with /docs/manage/ingestion-volume/ingest-budgets/index.md, where there's significant overlapping content. -->
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+:::note
+This feature is in Beta. To participate, contact your Sumo Logic account executive.
+:::
+
+Ingest Budgets Minute Volume, as the name suggests, lets you control data ingestion to minute granularity. This granular control in data ingestion protects against Ingestion surges and spikes thus safeguarding your account against throttling.
+
+Log data can be assigned to an ingest budget that defines a minute log capacity limit. The capacity is tracked based on the combined volume from all sources of log data. When an ingest budget's capacity is reached, you can have Sumo Logic stop collecting the log data assigned to it to safeguard against data volume spikes.
+
+An ingest budget's capacity usage is logged in the Audit Index when the audit threshold is reached and continues to be logged until the budget is reset.
+
+## Rules
+
+* Metrics and Traces data is not supported.
+* Fields assigned with Field Extraction Rules are not supported in the scope of an Ingest Budget.
```

Comment:
```suggestion
* Fields assigned with Field Extraction Rules are not supported in the scope of an ingest budget.
```

Reply:
@jpipkin1 **Ingest Budget** is a UI element. I'm wondering if we should capitalize that throughout, or leave it lowercased here since it's not referring to the UI element?

Reply:
@kimsauce, I feel that it needs to be lowercased if it doesn't refer to the UI element.